examples/material_gallery/lib/demo/fitness_demo.dart

```diff
@@ -0,0 +1,486 @@
+
+  Widget build(BuildContext context) {
+    if (!assetsLoaded) {
+      return new Scaffold(
+        toolBar: new ToolBar(
+          center: new Text("Fitness")
+        )
+      );
+    }
+
+    return new Scaffold(
+      toolBar: new ToolBar(
+        center: new Text("Fitness")
+      ),
+      body: new Material(
+        child: new Column(
+          justifyContent: FlexJustifyContent.center,
+          children: <Widget>[
+            _createWorkoutAnimation(),
+            _createWorkoutText(),
+            _createInfoPanel(),
+            _createWorkoutButton()
+          ]
+        )
+      )
+    );
+  }
+
+  Widget _createWorkoutAnimation() {
+    return new Flexible(
+      child: new Container(
+        decoration: new BoxDecoration(backgroundColor: Colors.grey[800]),
+        child: new SpriteWidget(workoutAnimation, SpriteBoxTransformMode.scaleToFit)
+      )
+    );
+  }
+
+  Widget _createWorkoutText() {
+    return new Padding(
+      padding: new EdgeDims.only(top: 20.0),
+      child: new Text("JUMPING JACKS", style: Theme.of(context).text.title)
+    );
+  }
+
+  Widget _createInfoPanel() {
+    return new Padding(
+      padding: new EdgeDims.only(top: 20.0, bottom: 20.0),
+      child: new Row(
+        justifyContent: FlexJustifyContent.center,
+        children: <Widget>[
+          _createInfoPanelCell("action/accessibility", "${workoutAnimation.count}", "COUNT"),
+          _createInfoPanelCell("image/timer", _formatSeconds(workoutAnimation.seconds), "TIME"),
+          _createInfoPanelCell("image/flash_on", "${workoutAnimation.kcal}", "KCAL")
+        ]
+      )
+    );
+  }
+
+  Widget _createInfoPanelCell(String icon, String value, String description) {
+    Color color;
+    if (workoutAnimation.workingOut)
+      color = Colors.black87;
+    else
+      color = Theme.of(context).disabledColor;
+
+    return new Container(
+      width: 100.0,
+      child: new Center(
+        child: new Column(
+          children: <Widget>[
+            new Icon(icon: icon, size: IconSize.s48, color: color),
+            new Text(value, style: new TextStyle(fontSize: 24.0, color: color)),
+            new Text(description, style: new TextStyle(color: color))
+          ]
+        )
+      )
+    );
+  }
+
+  Widget _createWorkoutButton() {
+    Color color;
+    String text;
+    VoidCallback onPressed;
+
+    if (workoutAnimation.workingOut) {
+      color = Colors.red[500];
+      text = "STOP WORKOUT";
+      onPressed = endWorkout;
+    } else {
+      color = Theme.of(context).primaryColor;
+      text = "START WORKOUT";
+      onPressed = startWorkout;
+    }
+
+    return new Padding(
+      padding: new EdgeDims.only(bottom: 10.0),
+      child: new SizedBox(
+        width: 300.0,
+        height: 72.0,
+        child: new RaisedButton (
+          onPressed: onPressed,
+          color: color,
+          child: new Text(
+            text,
+            style: new TextStyle(color: Colors.white, fontSize: 18.0)
```

Comment:
The color is not set by changing the color of the button and the ButtonTheme doesn't provide enough options to make the button look right in this context.